key2pdf.js

```diff
@@ -21,7 +22,7 @@ var parse = require('date-fns/parse');  //https://github.com/date-fns/date-fns
 var format = require('date-fns/format');  //https://github.com/date-fns/date-fns
 var differenceInMilliseconds = require('date-fns/difference_in_milliseconds'); //https://github.com/date-fns/date-fns
 var jobs = [];
-
+var gDriveUpload = require('./gdrive');
 
```

Comment:
Yep, exactly. Node will attempt those extensions automatically as part of the `require()` statement, so a lot of packages leave them off. I prefer leaving them off so it's a bit easier to read, so that's what I did in my modules, but I see you used the extensions in yours. Either way works.